signals_unix.go

```diff
@@ -24,9 +24,14 @@ import (
 	"strings"
 	"syscall"
 
+	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
 
+// SIGRTMIN is the value of SIGRTMIN, which depends libc.
+const SIGRTMIN = 34
```

Comment:
Note you are trying to set this from the makefile but you cannot do that with a `const`.
However I think this should be a static value anyway since containerd isn't doing anything except forwarding it along. Having this value change depending on what the host userspace is linked against doesn't really make sense since it is the container process that has to process the signal.

It should probably also just be a copy of what Docker does since this is going to be the expectation from build->deploy

Reply:
Thanks. I've removed the Makefile part from this PR.